content/zh/docs/unified_scheduling.md

```diff
@@ -0,0 +1,263 @@
++++
+title = "统一调度"
+date = 2024-12-30
+lastmod = 2024-12-30
+draft = false
+toc = true
+type = "docs"
+
+[menu.docs]
+  parent = "features"
+  weight = 3
++++
+
+## 概述
+
+Volcano 作为业界领先的云原生批处理系统调度器，通过统一的调度系统实现了对所有工作负载类型的支持：
+
+- 强大的批处理调度能力：通过 VcJob 完美支持 TensorFlow、PyTorch、Ray、Spark 等主流 AI 和大数据框架
+- 完整的 Kubernetes 工作负载支持：可直接调度 Deployment、StatefulSet、Job、DaemonSet 等原生工作负载
+
+这种统一调度能力让用户能够使用单一调度器来管理所有类型的工作负载，极大简化了集群管理复杂度。
+
+## 兼容Kubernetes调度能力
+
+### 1. 节点过滤机制 (Predicates)
+Volcano完整实现了Kube-Scheduler中的核心节点过滤机制,包括：
+
+- 基础资源过滤：节点可调度性、Pod 数量限制等
+- 亲和性/反亲和性：节点亲和性、Pod 间亲和性等
+- 资源约束：节点端口、存储卷限制等
+- 拓扑分布：Pod 拓扑分布约束等
+
+除了兼容 Kubernetes 的过滤器外，Volcano 还提供了以下增强特性：
+
+#### 节点过滤结果缓存 (PredicateWithCache)
+为了提升调度性能，Volcano 实现了节点过滤结果的缓存机制：
+
+##### 工作原理
+当调度器为 Pod 选择节点时，需要对每个节点进行一系列检查（如资源是否足够、是否满足亲和性要求等）。
+这些检查结果可以被缓存下来：
+
+- 如果短时间内要调度一个完全相同配置的 Pod，可以直接使用之前的检查结果
+- 避免重复进行相同的节点过滤计算
+- 显著提升批量创建 Pod 时的调度性能
+
+##### 配置方法
+在 volcano-scheduler-configmap 中启用缓存：
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: volcano-scheduler-configmap
+  namespace: volcano-system
+data:
+  volcano-scheduler.conf: |
+    actions: "enqueue, allocate, backfill"
+    tiers:
+    - plugins:
+      - name: predicates
+        arguments:
+          predicate.CacheEnable: true        # 启用节点过滤结果缓存
+```
+
+##### 使用场景
+1. 批量创建相同配置的 Pod
+   - 例如：创建多个相同的 TensorFlow 训练任务
+   - 第一个 Pod 完成节点过滤后，后续 Pod 可直接使用缓存结果
+
+2. 大规模集群调度优化
+   - 适用于 100+ 节点的集群
+   - 可显著减少调度延迟
+
+> **注意**：
+>
+> - 只有静态的检查结果才会被缓存（如节点标签、污点等）
+> - 动态资源相关的检查（如 CPU、内存用量）每次都会重新计算
+> - 当节点状态发生变化时，相关缓存会自动失效
+
+#### 资源比例预留 (ProportionalPredicate)
```

Comment:
This part is not a key feature and we can remove it to `Plugins` part.

Reply:
deleted